frontend/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,16 @@
+import { Typography, Box } from '@material-ui/core';
+import useStyles from './styles'
+
+
+const Footer = ({text=`Copyright \u00a9 TourDeFoos 2021`}) => {
```

Comment:
Nie ma testu na wartośc defaultową.

Reply:
To miał być właśnie test na wartość defaultową, ale coś mnie zaćmiło. Już poprawiam :D